hmi.c

```diff
@@ -181,46 +205,19 @@ float hmi_configure_force(float minimum_force, float force_step)
 	uint8_t force = 0;
 
```

Comment:
button_user_wait_action() already includes a while(1), remove this one.

Reply:
Comment bellow the line. :cry:

Actually, it is correct this way. We wait for user input (`button_user_wait_action()` loop), but then we are in another loop which allows us to increase the configured maximum force while running.

Reply:
I don't like so many while(1)... but yep, now I understand your point.